### PR TITLE
distinguishing another 404 error in autoscaling_gce_client.go

### PR DESCRIPTION
**Pull request by jayantjain93**

### CODE REVIEW

File: cluster-autoscaler/cloudprovider/gce/autoscaling_gce_client.go [hunk 1]
```diff
@@ -167,6 +167,11 @@ func (client *autoscalingGceClientV1) FetchMigTargetSize(migRef GceRef) (int64,
 func (client *autoscalingGceClientV1) FetchMigBasename(migRef GceRef) (string, error) {
 	registerRequest("instance_group_managers", "get")
 	igm, err := client.gceService.InstanceGroupManagers.Get(migRef.Project, migRef.Zone, migRef.Name).Do()
+	if err, ok := err.(*googleapi.Error); ok {
```
Comment (brett-elliott) [inline cluster-autoscaler/cloudprovider/gce/autoscaling_gce_client.go:170]: nit: you can make this
if err, ok := err.(*googleapi.Error); ok && err.Code == http.StatusNotFound {
...
}

Comment (jayantjain93) [reply cluster-autoscaler/cloudprovider/gce/autoscaling_gce_client.go]: Done

**Pull request merged**